language/compiler/src/main.rs

```diff
@@ -25,9 +25,6 @@ struct Args {
     /// Account address used for publishing
     #[structopt(short = "a", long = "address")]
     pub address: String,
-    /// Do not automatically compile stdlib dependencies
-    #[structopt(long = "no-stdlib")]
-    pub no_stdlib: bool,
```

Comment:
Ah, I modified the README for this directory literally yesterday, and it's already going to be out of date with this change 😅 If you can, maybe you could update `language/compiler/README.md`? I found it handy as a newcomer to the project in learning how to compiler MVIR.

Reply:
Yeah I have to admit that we haven't done an exceptional job keeping these docs up to date. For this one, let's wait and see if we decide to delete the crate as a whole. If not, I'll get the README updated.